lib/travis/build/git/clone.rb

```diff
@@ -17,6 +17,53 @@ def apply
         end
 
         private
+          TRACE_COMMAND_GIT_TRACE = "GIT_TRACE=true"
+          TRACE_COMMAND_STRACE = "strace"
+          DEFAULT_TRACE_COMMAND = TRACE_COMMAND_GIT_TRACE
+
+          def repo_slug
+            data.repository[:slug].to_s
+          end
+
+          def owner_login
+            repo_slug.split('/').first
+          end
+
+          def trace_git_commands_owners
+            ENV["TRACE_GIT_COMMANDS_OWNERS"].to_s.split(',')
```

Comment:
I think it is better to put this logic in `Travis::Build.config` (via `lib/travis/build/cofig.rb`).